api.swagger.yaml

```diff
@@ -6939,6 +6993,57 @@ components:
           type: object
           allOf:
             - $ref: '#/components/schemas/file-schema'
+    manual-page-listing-schema:
+      type: object
+      properties:
+        uuid:
+          description: Invotra UUID of the manual page.
+          type: string
+          format: uuid
+          example: '01234567-89ab-cdef-1234-56789abcdef0'
+        title:
+          description: Title of the manual page.
+          type: string
+          example: Code of Conduct
+        parent_pages:
+          description: An array of the manual page's parents.
```

Comment:
In what order? starting from the immediate parent, ending at the top-level ancestor, or vice versa?

Reply:
Order is now specified.